Makefile

```diff
@@ -241,7 +241,7 @@ integration_all: $(MINISHIFT_BINARY)
 	--run-before-feature="$(RUN_BEFORE_FEATURE)" --test-with-specified-shell="$(TEST_WITH_SPECIFIED_SHELL)" --copy-oc-from="$(COPY_OC_FROM)" $(GODOG_OPTS)
 
 .PHONY: integration_pr ## Run integration tests for pull request (skip more specialized tests)
-integration_pr: GODOG_OPTS = --tags=core\&\&~disabled
+integration_pr: GODOG_OPTS = --tags=addon-che,addon-che-with-anyuid
```

Comment:
@agajdosi is it required for this PR or generic now?

Reply:
@praveenkumar as discussed with @agajdosi I will remove this while merging this PR.